lang/cs/Org.Apache.REEF.Wake/Remote/Impl/StreamingRemoteManager.cs

```diff
@@ -118,6 +115,56 @@ public IObserver<T> GetRemoteObserver(IPEndPoint remoteEndpoint)
         }
 
         /// <summary>
+        /// Returns an IRemoteObserver used to send messages to the remote host at
+        /// the specified IPEndpoint.
+        /// </summary>
+        /// <param name="remoteEndpoint">The IPEndpoint of the remote host</param>
+        /// <returns>An IRemoteObserver used to send messages to the remote host</returns>
+        public IRemoteObserver<T> GetUnmanagedRemoteObserver(RemoteEventEndPoint<T> remoteEndpoint)
```

Comment:
I don't think it's a good idea to make it `internal` or `[Private]`, given that users can choose to use the wake library independently of REEF.